docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Additional database version information
+{: .no_toc }
+
+{% include page-toc.md %}
+
+
+## Where can I find my database version?
+
+### Option 1: View in your list of databases
+1. Select databases on the sidebar
+2. View your database's version under the version column
+
+### Option 2: View in your database's summary
+1. Select databases on the sidebar
+2. Click on your database's name
+3. View your database's version in the summary
+
+## Why is my database's version "Unknown"?
```

Comment:
An alternative view is to create a NOTE callout before the two methods of viewing the cloud db version which might look like this:

```
{: .note}
A database that fails to respond has an **unknown** version. [Learn more about unknown database versions](/docs/cloud/troubleshooting/issue-cloud-version-unknown)
```

This is preferable because it gives the user information up-front, so they're prepared in case this occurs. And they know where to find more info.

You would also add a "Further information" heading at the end of the file with the link to the troubleshooting issue